OpenSim/Common/Component.h

```diff
@@ -627,6 +627,12 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
      * and, therefore, calling this method is not "free". */
     std::string getAbsolutePathName() const;
 
+    /** Return a ComponentPath of the absolute path of this Component. 
+     * Note that this has more overhead than calling `getName()` because 
+     * it traverses up the tree to generate the absolute pathname (and 
+     * is thus a function of depth). Consider other options if this is
```

Comment:
`other options` -> `getAbsolutePathName()`.

Reply:
If we stick with this wording (rather than copying from function above), then this change may be misleading since both methods have to traverse the tree (even if `getAbsolutePathName()` is still faster).